index.d.ts

```diff
@@ -12,7 +12,8 @@ export interface Client {
     showProfileEditor(options: any): Promise<void>,
     showSocialAccounts(options: any): Promise<void>,
     showSocialLogin(options: any): Promise<void>,
-    showWebAuthnDevices(options: any): Promise<void>
+    showWebAuthnDevices(options: any): Promise<void>,
+    showMfa(options: any): Promise<void>
```

Comment:
Is there a notion of how this widget will work, e.g. to select email or phone number registration display? Will it also offer to stepup?

Reply:
I focused on the email registration. I have an idea on how to display the phone number and email registration (I thought  about displaying both fields). But I will tackle the subject when I will take the phone number registration ticket. Regarding the step up, I did not think about it but for now there is no ticket about adding the step up to the identity ui.